### ISSUE TITLE
OpenSWATH: Process killed when loading data file

### ISSUE THREAD

**Issue by SWATHnewbie** [OpenMS/OpenMS#4078]
Hello,

I apologizing in advance, I am incredibly new to both OpenSWATH and anything related to informatics in general.

After .mzML conversion my data is typically 20-40 GB big. When trying to run OpenSWATH on the data, depending on some settings (-threads for OpenSWATHworkflow and CPU selected in Docker), after a long time waiting at the step "Progress of 'Loading data file" I either get a message "Killed" or "ERRO[2847] error waiting for container: EOF". In the latter case the docker container shuts down. 

Example here: 

```
Loaded 28068 proteins, 409062 compounds with 2454372 transitions.
Loading mzML file FILE.mzML using readoptions cache
Progress of 'Loading metadata file FILE.mzML':
Will analyze the metadata first to determine the number of SWATH windows and the window sizes.
Determined there to be 64 SWATH windows and in total 2594 MS1 spectra
Determined there to be 64 SWATH windows and in total 2594 MS1 spectra
-- done [took 01:41 m (CPU), 06:21 m (Wall)] --
Progress of 'Loading data file FILE.mzML':
ERRO[3013] error waiting for container: EOF
```

Having previously used the provided .mzml data (~ 5GB) from the tutorial file (PASS00779) successfully, I know that technically the software is working fine. 
 
I suppose the file size could be the problem then. Are my PC requirements not sufficient? What minimum requirement would you set? My current Docker settings are 6 CPU with 10 GB RAM. Are there workarounds to make this work? 

In case it is of interest, here is my input: 

`OpenSwathWorkflow -in FILE.mzML -tr LIBRARY_decoy.pqp -tr_irt iRTassays.TraML -swath_windows_file SWATHwindows_analysis_64.tsv -batchSize 1000 -mz_extraction_window 0.05 -rt_extraction_window 600 -extra_rt_extraction_window 100 -Scoring:stop_report_after_feature 5 -out_osw FILE.osw -threads 4`

I've tried playing around with the "-threads" as well to no avail. 

I'd really appreciate any help I can get.

Thank you very much!
Sanni.

**Comment by grosenberger** [comment]
Hi Sanni,

The Docker container has access to about 10GB RAM, but just loading such big files requires even more storage. What you should do here is to enable caching by adding these parameters to the ``OpenSwathWorkflow`` command:

```
-readOptions cacheWorkingInMemory -tempDirectory /tmp/
```

Best regards,
George

**Comment by SWATHnewbie** [comment]
Hey George,

thank you very much for you fast answer! I actually had tested that before and sadly it did not resolve my issue.

However, I did end up finding the solution to my admittedly stupid problem just now which ended up being the low amount of available disk space on the drive I had docker/the VM running. I just changed that and now it appears to be running smoothly. Maybe this helps someone else in the future. 

Thank you again !

Best regards,
Sanni.
